### ISSUE TITLE
Investigate more advanced outdoor cameras.

### ISSUE THREAD

**Issue by tfabris** [tfabris/CrowCam#3]
Brands to look at:
- Axis
- Reolink
- Others

Some resources:

How to live stream:
https://birdwatchinghq.com/stream-live-animal-cam/

Example of someone with a similar setup (the person who wrote the how-to above):
https://www.youtube.com/watch?v=QgcN9_cqMP8

Camera:
M2026-LE Mk II   $380.55
https://www.amazon.com/Axis-Communications-01049-001-Outdoor-Network/dp/B077KV7Z68/ref=sr_1_1

Investigate Reolink PTZ (pan tilt zoom) cameras, some are supported by Surveillance Station such as this Reolink RLC-423 (which has a jack for external audio but not built in):
https://www.amazon.com/dp/B016UCNP08/ref=as_li_ss_tl

Reolink RLC-420 (Not PTZ):
https://www.amazon.com/Reolink-Security-Megapixels-2560x1440-Detection/dp/B01B4BKJOK/ref=sr_1_1_sspa
RLC-420 WAS MY ORIGINAL CAMERA - It failed after one year.
Note: Angle of View Horizontal: 80°, Vertical: 42°

Seems like all Reolink cameras except the battery powered ones like Argus should theoretically be compatible with Synology:
https://support.reolink.com/hc/en-us/articles/360004124293-Add-Reolink-Cameras-to-Synology-Surveillance-Station

**Comment by tfabris** [comment]
The prior camera forced my hand: It failed after about a year. Researching a new one now.

**Comment by tfabris** [comment]
Investigations so far:
- All the oudoor dome cameras I could find with PTZ seem to need an external microphone.
- Those with built-in mics are not the dome type, and I want the non-dome types to keep the raccoons from climbing onto the cameras.
- Example of one with PTZ, mics, everything, BUT, it's a raccoon-climber mounting system, so, no-go: https://www.amazon.com/Microphone-SV3C-Security-Surveillance-Recording/dp/B081Q47ZZ7

This one claims to be an outdoor PTZ dome camera with built-in microphone:
- Seller sites:
 https://www.cctvcameraworld.com/ip-ptz-mini-mic-night-vision.html
http://www.123-cctv.com/mini-ptz-camera-3x-zoom-mic-ir.html
- SKU is the same at both those sites: IPSDIR3X - But I can't tell if that's the model number or what. Can't find a brand on those.
- Can't find that IPSDIR3X camera above on Amazon. Since the sites above do not give a model number it's hard to track down.
- Similar looking cameras, but which do not have built in microphones:
https://www.amazon.com/LINOVISION-DH-SD1A203T-GN-2-7-8-1mm-Upgradable-Plug-n-Play/dp/B07XZGQJ8N/
https://www.dahuasecurity.com/asset/upload/product/20180522/SD1A203T-GN_Datasheet_20180522.pdf
- I'm having qualms about that camera because the sites look kinda fly-by-night and I can't see the brand and model of the cameras, so I can't independently investigate them from other places.

There are some dome cameras with PTZ and external mic connections. You do NOT want these:
- External mics require more wiring, meaning possibly weatherproofing problems.
- External mics require additional power. For instance the Hikvision below has bare wires for mic, but you have to supply power yourself.
- Examples:
https://www.amazon.com/Weatherproof-Sensitive-Microphone-Security-Surveillance/dp/B07FL1WDC5
https://www.microseven.com/product/IP-Cameras-M7WP-MIC.html
- This external mic is a hikvision mic to connect to security cameras, can't even tell if it's weatherproof, but when you look at the wiring diagram it needs power: https://www.amazon.com/Hikvision-Microphone-DS-2FP2020-Security-Sensitive/dp/B07ZT5229B/
- Perhaps the better choice is to go with a better stationary camera with a built in mic.
- The Hikvision web site says that some of their models (a small hard-to-find number of them) do have a power output for external mics, but I think that's too much trouble now.

Example PTZF (remote pan tilt zoom focus) camera I was interested in until I saw the audio wiring problems:
- Hikvision DS-2DE2A404IW-DE3
- PoE 802.3af, PTZF, Outdoor, external audio plugin "1-ch audio input and 1-ch audio output" says datasheet.
- 24 fps (2560 × 1440), 30 fps (2048 × 1536, 1920 × 1080, 1280 × 960, 1280 × 720)
- Horizontal field of view: 100° to 33° (wide-tele),
- Synology compatible
- Multiple amazon sellers (examples):
https://www.amazon.com/DS-2DE2A404IW-DE3-2-8mm-12mm-Vision%EF%BC%8C2560X1440%EF%BC%8CAudio-Interface%EF%BC%8CSmart-Recording%EF%BC%8CIP66/dp/B07Q2V5CDP/
https://www.amazon.com/OEM-DS-2DE2A404IW-DE3-2-8mm-12mm-4-Megapixel-2560x1440/dp/B07HP1JNTX/\
- Datasheet: https://www.hikvision.com/mtsc/uploads/product/accessory/Datasheet_of_DS-2DE2A404IW-DE3(2.8-12MM)_20180328(1).pdf

Note: My current POE switch is:
- Amcrest Gigabit Uplink 9-Port Poe+ Ethernet Switch 802.3at:
https://www.amazon.com/gp/product/B073V4QRXQ/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1
https://amcrest.com/amcrest-gigabit-9-port-poe-ethernet-switch-with-8-ports-poe-plus-802-3at-96w-sfp-optical.html
- Manual and Datasheet:
https://s3.amazonaws.com/amcrest-files/Documentation/AGPS9E8P-AT-96+User+Manual.pdf
https://s3.amazonaws.com/amcrest-files/Documentation/AGPS9E8P-AT-96+Technical+Specifications.pdf

Note: Difference between 802.3at and 802.3af: https://blog.twinstate.com/poe-vs-poe-plus
- Basically, "at" is better than "af". "at" carries the "PoE+" designation
- 802.3af (PoE) can deliver 15.4 watts over Cat5 cables.
- 802.3at (PoE+) can deliver up to 30 watts over Cat 5 cables with 25.5 watts available to devices.

**Comment by tfabris** [comment]
Axis M50 series is PTZ with built in mic, however, they are all indoor only.
https://www.axis.com/en-us/products/axis-m5055

Axis P33 series has built in mics, but all the ones rated for outdoors do not have mics.
https://www.axis.com/en-us/products/axis-p33-series

Axis M5525–E PTZ has external microphone input. Maybe Axis mics are easier to set up?
https://www.axis.com/files/datasheet/ds_m5525e_t10104238_en_1902.pdf

**Comment by tfabris** [comment]
This one looks similar to the other outdoor PTZ/Mic camera "IPSDIR3X" from those other two sites, however, it does not seem to have a microphone, so I still need to find out how to tell if the "IPSDIR3X" really has a microphone.
Linovision SD1A203T-GN (OEM version of Dahua DH-SD1A203T-GN)
https://www.amazon.com/LINOVISION-DH-SD1A203T-GN-2-7-8-1mm-Upgradable-Plug-n-Play/dp/B07XZGQJ8N
Spec sheet: https://images-na.ssl-images-amazon.com/images/I/B1NGrsX4afS.pdf

**Comment by tfabris** [comment]
Dahua N53AJ5Z
Varifocal (but not PTZ, no zoom, Outdoor, built in microphone:
https://us.dahuasecurity.com/product/5mp-vari-focal-starlight-eyeball-with-smart-motion-detection/
Datasheet: https://us.dahuasecurity.com/wp-content/uploads/2020/01/Datasheet_5MP_Starlight_SMD_Varifocal_Eyeball_Network_Camera_Arctic_N53AJ5Z_001.001.pdf
(2592 x 1944) at 20 fps, 4 MP (2688 x 1520) at 30 fps
Highly likely to be Synology Compatible, since there are many N5-series cameras and a "generic dahua" on the compatibilty list.

**Comment by tfabris** [comment]
Fixed lens cameras with built in mics (choice between 2.8mm and 3.6mm lens ):
Dahua N44CG52 / N44CG53
https://us.dahuasecurity.com/product/4mp-ir-3-6mm-epoe-eyeball_2_8/
https://us.dahuasecurity.com/product/4mp-ir-fixed-lens-epoe-eyeball-3_6/
Supposedly outdoor (the spec sheet doesn't say outdoor but does say waterproof and "arctic pro").
Main Stream: 4 MP at 30 fps
 PoE (IEEE 802.3af)
Synology compatible (on the list).
Datasheet for both: https://us.dahuasecurity.com/wp-content/uploads/2019/06/Datasheet_4MP_FixedEyeball_NetworkCamera_ePoE_Arctic_N44CG52_N44CG53_v001.004.pdf

**Comment by tfabris** [comment]
One of the fly-by-night looking web sites (cctv camera world), which doesn't print brand names or model numbers, has a feature selector:
https://www.cctvcameraworld.com/poe-ip-security-cameras/dome_camera-built_in-yes,_ip67-yes,_ip66.html

From this we get this list:
Avalonix (model number unknown) 8MP 4K IP Dome Camera, Built-in Mic, 4X Zoom $330.00
https://www.cctvcameraworld.com/4k-dome-ip-camera-audio-4x.html
Can't get details/specs from Avalonix web site, it's "dealers only".
The brand name "Avalonix" is not on the Synology compatibility list. However the Q+A section says that it is "ONVIF Compliant" which is a CCTV camera interoperability standard, and the Synology compatibility list does indeed have "Generic_ONVIF" in the list. I asked a question from the web site, and in minutes, got a response saying "yes it is Synology compatible via ONVIF driver".
2.7-12mm Motorized Zoom Lens Lens controllable from NVR, Smartphone or Computer (not PTZ, just Z)
Main Stream: 4K (1-15fps), 3MP (1-30fps)
Close Focus Distance	12 inches
Angle of View	Horizontal:102°-39°, Vertical:55°-22°
PoE (802.3af)
Outdoor IP67
4.8" x 4.2" (diameter x height)

Positive points:
- The close focus distance of 12" is perfect. I think our distance to the feeder is in the neighborhood of two feet, so close focus is very useful.

Concerns:
- Does varifocal mean that if I focus the camera on the feeder, that the background is out of focus? 
- Does it auto focus to the thing that is having motion on the screen?

**Comment by tfabris** [comment]
Amcrest (same brand as my PoE switch) cameras in this category:

Fixed 2.8mm lens:
Amcrest
White: IP8M-T2499EW-28MM
https://amcrest.com/ultrahd-4k-8mp-turret-poe-ip-camera-security-3840x2160-98ft-nightvision-2-8mm-lens-112-ip67-weatherproof-microsd-recording-white-ip8m-t2499ew.html
 Black: IP8M-T2499EB-28MM
https://amcrest.com/ultrahd-4k-8mp-turret-poe-ip-camera-security-3840x2160-98ft-nightvision-2-8mm-lens-112-ip67-weatherproof-microsd-recording-black-ip8m-t2499eb-28mm.html
Tech Specs:
https://drive.google.com/file/d/11bIYRP3R8b6kjzDgDsoyYgeQuTRdtei-/view
Angle of View H: 112°, V: 69°
Close Focus Distance 0.9m (2.95ft)
PoE (802.3af)
Outdoor IP67

**Comment by tfabris** [comment]
Dahua IPC-HDW5831R-ZE
8MP WDR IR Eyeball Network Camera
https://www.dahuasecurity.com/products/productDetail/14301?us
Datasheet:
https://www.dahuasecurity.com/asset/upload/uploads/soft/20181227/DH-IPC-HDW5831R-ZE_Datasheet_20181227.pdf
Synology compatible (directly supported)
Built-in Mic 
Outdoor: IP67
Close Focus Distance 12 inches
Angle of View H:110°~40° , V:58°~23°
PoE(802.3af)
15fps@4K(3840×2160),25/30fps@3M(2304×1296)

**Comment by tfabris** [comment]
Dahua camera is working pretty well for now. Closing this ticket.
